src/Config/Converters/Utf8JsonReaderExtensions.cs

```diff
@@ -1,11 +1,12 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using System.Text;
 using System.Text.Json;
 
 namespace Azure.DataApiBuilder.Config.Converters;
 
-static internal class Utf8JsonReaderExtensions
+static public class Utf8JsonReaderExtensions
```

Comment:
why are we making public? To be used in tests? Cant we do internals visible to test project?

Reply:
no, it's also being called up by RuntimeCongigValidator